docs/api.yaml

```
@@ -24,6 +24,59 @@ paths:
       responses:
         '201':
           description: Person added
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Person"
+        '400':
+          description: Missing required fields
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: "Missing required field(s): first_name, slack_user_id"
+        '409':
+          description: slack_user_id conflict
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: "Person with slack_user_id = 'W012A3CDE' already exists"
+
+  /people/{slack_user_id}:
+    get:
+      summary: Returns a specific Person by their slack_user_id with quotes in Nostalgiabot
+      parameters:
+        - in: path
+          required: true
+          name: slack_user_id
+          description: The unique ID assigned by Slack to a user
+          schema:
+            type: string
+            example: "W012A3CDE"
+      responses:
+        "200":
+          description: "Nostalgiabot remembers this person"
```

Lol I know I wrote the one for GET all people, but I feel like backtracking on this phrasing. I think we should just keep the API sterile like in your other specs and just have the end result Slack messages with the more humanistic tone

```suggestion
          description: "Person matching criteria:"
```

And with this change, do you mind also changing the one for GET (all) people?

---

Haha no problem! I agree. I was just following what already existed, but I like this suggestion better 😄